adafruit_platformdetect/board.py

```diff
@@ -68,6 +68,8 @@
 
 MICROCHIP_MCP2221           = "MICROCHIP_MCP2221"
 
+BINHO_NOVA                  = "NOVA"

```

Comment:
My only concern on this PR is that there's another board named Nova that will be added in the future. Can we change the string from `"NOVA"` to `"BINHO_NOVA"`? Otherwise, everything else looks good to me too.

Reply:
Yeah, sure. I can make this change to use "BINHO_NOVA" instead. Thanks for the heads up!